traceapp/vis.go

```diff
@@ -92,6 +92,11 @@ func (a *App) d3timelineInner(t *appdash.Trace, depth int) ([]timelineItem, erro
 	}
 	for _, e := range events {
 		if e, ok := e.(appdash.TimespanEvent); ok {
+			// Continue to next iteration
+			// if e.Start() or e.End() are empty time values.
+			if e.Start() == (time.Time{}) || e.End() == (time.Time{}) {
+				continue
+			}
```

Comment:
Instead of continuing (not showing this timespan) we should choose arbitrary times here.

For example, set `start` to `time.Now()` and `end` to `end.Add(1 * time.Millisecond)` (confirm this works OK in the frontend web UI, the value may need to be larger)